scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -213,31 +211,103 @@ task LoadData {
     }
   }
 
+  ## TODO what is external_sample_names!?!?!??! Can I swap to sample_names for now?!?!
```

Comment:
`external_sample_names` is an array of strings that is usually passed from the Terra Data and becomes the values for `sample_info.sample_name`